### PR TITLE
Add example for literalpath to Remove-Item

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [x] Impacts 6.0 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [x] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/3.0/Microsoft.PowerShell.Management/Remove-Item.md [hunk 1]
```diff
@@ -80,6 +80,43 @@ The path is specified, but the optional parameter name (Path) is omitted.
...
+    Directory: C:\temp\Downloads
+
+Mode                LastWriteTime         Length Name
+----                -------------         ------ ----
+-a----         6/1/2018  12:19 PM           1362 myFile.txt
+-a----         6/1/2018  12:30 PM           1132 myFile[1].txt
+-a----         6/1/2018  12:19 PM           1283 myFile[2].txt
+-a----         6/1/2018  12:19 PM           1432 myFile[3].txt
+
+PS C:\temp\Downloads> Get-ChildItem | where name -like '*`[*'
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Management/Remove-Item.md]: Please update sample to not use alias and to proper case the variable and operator.

File: reference/4.0/Microsoft.PowerShell.Management/Remove-Item.md [hunk 2]
```diff
@@ -88,6 +88,42 @@ The path is specified, but the optional parameter name (Path) is omitted.
...
+    Directory: C:\temp\Downloads
+
+Mode                LastWriteTime         Length Name
+----                -------------         ------ ----
+-a----         6/1/2018  12:19 PM           1362 myFile.txt
+-a----         6/1/2018  12:30 PM           1132 myFile[1].txt
+-a----         6/1/2018  12:19 PM           1283 myFile[2].txt
+-a----         6/1/2018  12:19 PM           1432 myFile[3].txt
+
+PS C:\temp\Downloads> Get-ChildItem | where name -like '*`[*'
```
Comment (DCtheGeek) [inline reference/4.0/Microsoft.PowerShell.Management/Remove-Item.md]: Please update sample to not use alias and to proper case the variable and operator.

File: reference/5.1/Microsoft.PowerShell.Management/Remove-Item.md [hunk 3]
```diff
@@ -88,6 +88,42 @@ The path is specified, but the optional parameter name (*Path*) is omitted.
...
+    Directory: C:\temp\Downloads
+
+Mode                LastWriteTime         Length Name
+----                -------------         ------ ----
+-a----         6/1/2018  12:19 PM           1362 myFile.txt
+-a----         6/1/2018  12:30 PM           1132 myFile[1].txt
+-a----         6/1/2018  12:19 PM           1283 myFile[2].txt
+-a----         6/1/2018  12:19 PM           1432 myFile[3].txt
+
+PS C:\temp\Downloads> Get-ChildItem | where name -like '*`[*'
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Management/Remove-Item.md]: Please update sample to not use alias and to proper case the variable and operator.

File: reference/6/Microsoft.PowerShell.Management/Remove-Item.md [hunk 4]
```diff
@@ -86,6 +86,42 @@ The path is specified, but the optional parameter name (*Path*) is omitted.
...
+    Directory: C:\temp\Downloads
+
+Mode                LastWriteTime         Length Name
+----                -------------         ------ ----
+-a----         6/1/2018  12:19 PM           1362 myFile.txt
+-a----         6/1/2018  12:30 PM           1132 myFile[1].txt
+-a----         6/1/2018  12:19 PM           1283 myFile[2].txt
+-a----         6/1/2018  12:19 PM           1432 myFile[3].txt
+
+PS C:\temp\Downloads> Get-ChildItem | where name -like '*`[*'
```
Comment (DCtheGeek) [inline reference/6/Microsoft.PowerShell.Management/Remove-Item.md]: Please update sample to not use alias and to proper case the variable and operator.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/3.0/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/4.0/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/5.0/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/5.1/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/6/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C1%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806012005467944-2487%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=6kZ3dUXx%2Bod5aj%2F%2BVj7rbVZknwQVnIorlIByWmbd2hA%3D&st=2018-06-01T20%3A10%3A05Z&se=2018-07-02T20%3A15%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Management/Remove-Item.md [hunk 5]
```diff
@@ -86,6 +86,42 @@ The path is specified, but the optional parameter name (*Path*) is omitted.
 The *Recurse* parameter deletes all of the contents of the OldApp key recursively.
 If the key contains subkeys and you omit the *Recurse* parameter, you are prompted to confirm that you want to delete the contents of the key.
 
+### Example 6 - Deleting files with special characters
```
Comment (DCtheGeek) [inline reference/6/Microsoft.PowerShell.Management/Remove-Item.md]: Please consider changing this to a non-Windows example.

File: reference/3.0/Microsoft.PowerShell.Management/Remove-Item.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Management/Remove-Item.md]: Arrgh… I thought I have fixed that before I pasted.

File: reference/6/Microsoft.PowerShell.Management/Remove-Item.md (hunk 5 above)
Comment (sdwheeler) [reply reference/6/Microsoft.PowerShell.Management/Remove-Item.md]: Later. I don't have a test environment for that.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/3.0/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/4.0/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/5.0/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/5.1/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/6/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C1%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806012026582082-2487%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=BxX18y0EJ34jww7RBg9wS8XCTdLdPdaj1cHntVr67aA%3D&st=2018-06-01T20%3A31%3A10Z&se=2018-07-02T20%3A36%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/3.0/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/4.0/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/5.0/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/5.1/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Management/Remove-Item.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2143/reference/6/Microsoft.PowerShell.Management/Remove-Item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C1%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806012034147715-2487%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=rZ942BDTr0FXukkZH7taD8U%2FoUzEpry9GQLNwaEH7vs%3D&st=2018-06-01T20%3A40%3A06Z&se=2018-07-02T20%3A45%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**